yapapi/executor/__init__.py

```diff
@@ -280,6 +280,7 @@ def input_tasks() -> Iterable[Task[D, R]]:
         last_wid = 0
 
         agreements_to_pay: Set[str] = set()
+        debit_notes_accepted: Set[str] = set()
```

Comment:
We need a better name for this set...

Reply:
My proposal: `agreements_accepting_debit_notes`.